tests/TestCase/Model/Behavior/HashidBehaviorTest.php

```diff
@@ -109,6 +109,28 @@ public function testFind() {
 	/**
 	 * @return void
 	 */
+	public function testFindList() {
+		$this->Addresses->behaviors()->Hashid->config('field', 'hashid');
+		$this->Addresses->displayField('city');
+		$this->Addresses->primaryKey('id');
+
+		$data = [
+			'city' => 'Foo'
+		];
+		$address = $this->Addresses->newEntity($data);
+		$res = $this->Addresses->save($address);
+
+		$id = $address->id;
+		$hasher = new Hashids();
+		$hashid = $hasher->encode($id);
+
+		$addressList = $this->Addresses->find('list')->toArray();
```

Comment:
keyField/valueField in find(list) should be filled instead IMO. But it seems it also works this way.